extras/rya.console/src/main/java/org/apache/rya/shell/SharedShellState.java

```diff
@@ -23,10 +23,10 @@
 import java.util.Objects;
 import java.util.concurrent.locks.ReentrantLock;
 
-import javax.annotation.Nullable;
-import javax.annotation.ParametersAreNonnullByDefault;
-import javax.annotation.concurrent.Immutable;
-import javax.annotation.concurrent.ThreadSafe;
+import edu.umd.cs.findbugs.annotations.Nullable;
```

Comment:
Do you mean the new package name? Yeah, that's intentional but I don't know if it's correct.  I was told to use findbug-annotations, and that's where the nullable annotation lives.